src/lib/plugin.js

```diff
@@ -375,7 +377,7 @@ class FiveBellsLedger extends EventEmitter2 {
     try {
       res = yield request(Object.assign({
         method: 'get',
-        uri: creds.account,
+        uri: this.appendAuthToken(creds.account),
         json: true
       }, requestCredentials(creds)))
```

Comment:
`auth` seems to be for passing in basic auth credentials and does not seem to support to append an auth token to the query string.